### ISSUE TITLE
colon: "always" giving false positives when defining several selectors at once

### ISSUE THREAD

**Issue by satoriweb** [rossPatton/stylint#136]
Hi,

In Stylint 1.0.9 on Windows 7, Styling is giving warnings on @extends and when defining several selectors at once. 

Given; 
`"colons": "always",`

Examples;
```
Warning: missing colon between property and value
Line: 50: .amount, .info
```
```
Warning: missing colon between property and value
Line: 141: .account-totals__balance, .account-totals__balance--loading, .account-totals__balance--none
```
```
Warning: missing colon between property and value
Line: 14: @extend .collapse-link-heading:hover
```
```
Warning: missing colon between property and value
Line: 12: @extend .icon-chevron-right:before
```

Thanks!

**Comment by rossPatton** [comment]
Will take a look

**Comment by rossPatton** [comment]
@satoriweb This issue is resolved in develop, it will be released shortly
